.github/workflows/ftr-e2e-reporting-test.yml

```diff
@@ -0,0 +1,188 @@
+          echo "yarn_version=$(jq -r '.engines.yarn' package.json)" >> $GITHUB_OUTPUT
+        working-directory: OpenSearch-Dashboards
+        shell: bash
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ steps.tool-versions.outputs.node_version }}
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Setup Opensearch Dashboards
+        run: |
+          npm uninstall -g yarn
+          echo "Installing yarn ${{ steps.tool-versions.outputs.yarn_version }}"
+          npm i -g yarn@${{ steps.tool-versions.outputs.yarn_version }}
+          yarn cache clean
+        working-directory: OpenSearch-Dashboards
+        shell: bash
+
+      - name: OpenSearch Dashboards Plugin Bootstrap
+        run: |
+          max_retries=3
+          count=0
+          until yarn osd bootstrap; do
+            exit_code=$?
+            count=$(($count + 1))
+            echo "Attempt $count/$max_retries failed with exit code $exit_code, retrying..."
+            if [ $count -ge $max_retries ]; then
+              echo "Reached maximum retry attempts, stopping..."
+              exit $exit_code
+            fi
+            sleep 5 # Wait for 5 seconds before retrying
+          done
+        working-directory: OpenSearch-Dashboards
+        shell: bash
+
+      - name: Run Opensearch Dashboards with Dashboards Reporting Plugin Installed
+        run: |
+          nohup yarn start --no-base-path --no-watch | tee dashboard.log &
+        working-directory: OpenSearch-Dashboards
+
+      - name : Check If OpenSearch Dashboards Is Ready
+        if: ${{ runner.os == 'Linux' }}
+        run: |
+          if timeout 600 grep -q "bundles compiled successfully after" <(tail -n0 -f dashboard.log); then
+            echo "OpenSearch Dashboards compiled successfully."
+          else
+            echo "Timeout for 600 seconds reached. OpenSearch Dashboards did not finish compiling."
+            exit 1
+          fi
+        working-directory: OpenSearch-Dashboards
+
+      - name: Checkout Dashboards Functioanl Test Repo
+        uses: actions/checkout@v2
+        with:
+          path: opensearch-dashboards-functional-test
+          repository: opensearch-project/opensearch-dashboards-functional-test
+          ref: ${{ env.OPENSEARCH_DASHBOARDS_VERSION }}
+          fetch-depth: 0
+
+      - name: Install Cypress
+        run: |
+          npm install cypress --save-dev
+        shell: bash
+        working-directory: opensearch-dashboards-functional-test
+
+      - name: Get Cypress version
+        id: cypress_version
+        run: |
+          echo "::set-output name=cypress_version::$(cat ./package.json | jq '.dependencies.cypress' | tr -d '"')"
+        working-directory: opensearch-dashboards-functional-test
+
+      - name: Run Cypress tests
+        run: |
+          yarn cypress:run-without-security --browser chrome --spec 'cypress/integration/plugins/reports-dashboards/*.js'
```

Comment:
Do we want to consider using Chromium with headless mode?

Reply:
yes we can give a try.

Reply:
done

Reply:
After switch to chromium, the results remain the same.